static/scss/district-list.scss

```diff
@@ -45,6 +45,16 @@
   text-align: center;
 }
 
+.district-list-text {
+  font-size: 1.2em;
+}
+
+@media (max-width: 600px) {
```

Comment:
This is SCSS so you can nest the media query if you want.

Reply:
derp, of course